framework/components/ema-list-items/ema-list-items.twig

```diff
@@ -1 +1,41 @@
 {% include '@ec-europa/ecl-list-items' %}
+
+{% if variant == 'translation' %}
+  <section class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+    <div class="ecl-file__translations">
+      {% include '@ec-europa/ecl-buttons' with {
+        'label': translations_label,
+        'modifier': 'secondary',
+        'extra_classes': 'ecl-file__translations-toggle',
+        'extra_attributes': [
+          { 'name': 'aria-controls', 'value': 'translations-expand' },
+          { 'name': 'aria-expanded', 'value': 'true' },
+          { 'name': 'id', 'value': 'translations-expand-button' },
+          { 'name': 'type', 'value': 'button' },
+          { 'name': 'title', 'value': translations_tooltip }
+        ]
+      } %}
+
+      <div aria-hidden="false" aria-labelledby="translations-expand-button" id="translations-expand">
+        <ul class="ecl-file__translations-list">
+          {% if translations is defined %}
+            {% for translation in translations %}
+              <li class="ecl-file__translations-item">
```

Comment:
Border between translation items is much darker than the one between list items.
If possible, we should use the same shade of grey.